java/src/jmri/jmrix/loconet/SlotManager.java

```diff
@@ -53,13 +53,13 @@ public class SlotManager extends AbstractProgrammer implements LocoNetListener,
      * Time to wait after programming operation complete on LocoNet
      * before reporting completion and hence starting next operation
      */
-    static public int postProgDelay = 100; // this is public to allow changes via script
```

Comment:
Does this reduce the post-delay for _all_ programming operations?  Will this affect any CS or programming device?  Will this affect those non-mobile-decoder devices, including CV-programmed stationary devices or LocoNet-programmable devices devices?

Reply:
Only traditional CV reads and writes use this delay setting. It's in addition to the method-specific settings, for example the ones for programming-track and ops-mode operations just below.